src/components/ZetkinCalendar/index.tsx

```diff
@@ -67,6 +71,7 @@ const ZetkinCalendar = ({ baseHref, events, campaigns , tasks }: ZetkinCalendarP
         }
     };
 
+    const weekNumber = dayjs().isoWeek();
```

Comment:
Does this really give us the week number currently displayed in the calendar? My guess from just looking at the code is that it would always be the week number of the _current date_, meaning that it does not change as you move back and forth in the calendar.